src/endpoints.js

```diff
@@ -13,14 +13,15 @@ export const IPFS_ENDPOINT = isLocalOrUnknownNetwork(CHAIN_ID)
 
 // Court server endpoint
 export function courtServerEndpoint() {
-  if (isLocalOrUnknownNetwork(CHAIN_ID)) {
-    return 'http://127.0.0.1:8050'
-  }
-
-  const networkType = getNetworkType(CHAIN_ID)
-  return `https://court-backend${
-    networkType === 'main' ? '' : `-${COURT_SERVER_NAME || networkType}`
-  }.eth.aragon.network`
+  // if (isLocalOrUnknownNetwork(CHAIN_ID)) {
+  //   return 'http://127.0.0.1:8050'
+  // }
+
+  // const networkType = getNetworkType(CHAIN_ID)
+  // return `https://court-backend${
+  //   networkType === 'main' ? '' : `-${COURT_SERVER_NAME || networkType}`
+  // }.eth.aragon.network`
+  return 'https://court-backend-usability.eth.aragon.network'
```

Comment:
Reminder to update this (although maybe this is also a hint to provide an env var to control this)